Index.html

```diff
@@ -1,179 +1,283 @@
+            <img src="./Contents/Resources/Pictures/Geometry-bottom-left.svg" alt="" class="deco__bottomLeft">
+            <img src="./Contents/Resources/Pictures/Geometry-right.svg" alt="" class="deco__right">
+            <img src="./Contents/Resources/Pictures/GEometry_middle.svg" alt="" class="deco__middle">
+        </section>
+
+        <section id="about" class="about">
+            <article class="about__article">
+                 <h2 class="h2 about__h2">About</h2>
+                <p class="about__text">
+                    I Prefer To Speak By My Gestures
+                </p>
+
+                <button class="buttons button__getCV">
+                    Get my CV
+                    <span class="icon material-symbols-outlined">
+                        download
+                    </span>
+                </button>
+            </article>
+
+            <ul class="all">
+                <li class="languages about__lists">
+                    <h3 class="h3">Languages</h3>
+                    <ul class="about__ul">
+                        <li class="about__li">JavaScript</li>
+                        <li class="about__li">Python</li>
+                        <li class="about__li">HTML</li>
+                        <li class="about__li">CSS</li>
+                    </ul>
+                    <div class="about__icon">
+                        <img src="./Contents/Resources/Icons/Sharp.svg" alt="" class="pound__logo">
+                    </div>
+                </li>
+
+                <li class="frameworks about__lists">
+                    <h3 class="h3">Frameworks</h3>
+                    <ul class="about__ul">
+                        <li class="about__li">Bootstrap</li>
+                        <li class="about__li">Django</li>
+                        <li class="about__li">Rest</li>
+                        <li class="about__li">Celery</li>
+                    </ul>
+
+                    <div class="about__icon">
+                        <img src="./Contents/Resources/Icons/Rectangle 19.svg" alt="" class="square__logo">
+                    </div>
+                </li>
+
+                <li class="skills about__lists">
+                    <h3 class="h3">Skills</h3>
+                    <ul class="about__ul">
+                        <li class="about__li">Postgres</li>
+                        <li class="about__li">RabbitMQ</li>
+                        <li class="about__li">GitHub</li>
+                        <li class="about__li">Linux</li>
+                        <li class="about__li">Docker</li>
+                    </ul>
+
+                    <div class="about__icon">
+                        <img src="./Contents/Resources/Icons/ic_skills.svg" alt="" class="idea__logo">
+                    </div>
+                </li>
+
+                <img src="./Contents/Resources/Icons/image_dots_with_teal.svg" alt="" class="about__deco">
+            </ul>
```

Comment:
According to the Figma template, you should Follow the Design to make your project exactly like the Design, Kindly I suggest you follow this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=X7Yh3FuNhyLhpYDE-0) to make your project match the design template.

> ### The Figma Design
>
><img width="194" alt="Screenshot 2023-04-28 at 15 31 27" src="https://user-images.githubusercontent.com/91237525/235237297-30ca7512-49f3-48a1-81bc-2d3373fcfe27.png">
>
>
>### Your Design
>
><img width="188" alt="Screenshot 2023-04-28 at 15 31 44" src="https://user-images.githubusercontent.com/91237525/235237344-665fc81d-c0f2-441b-b798-119205231fe8.png">


Kindly, I suggest you use the font size, letter-spacing, margin, and padding as it is in the Figma template.